README.md

```diff
@@ -1,42 +1,160 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
-#page
+<div align="center">
 
+  <img src="./Images/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-## Built With
+</div>
 
-- HTML
-- CSS
-- JavaScript
-## Link
-https://eddyshimwa.github.io/Portforio/
+# 📗 Table of Contents
 
-## Link
-https://eddyshimwa.github.io/Portforio/
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-👤 **Author**
+# 📖 [Creative Commons Global Summit] <a name="about-project"></a>
+
+> 
+
+**[Creative Commons Global Summit]** Is a first capstone project in microverse, It includes all the lessons we learned in 4 weeks including HTML, javaScript, CSS, bootstrap 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">HTML</a></li>
+    <li><a href="">CSS</a></li>
+    <li><a href="">javaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Completed Mobile version]**
+- **[Completed Desktop version]**
+- **[Added functionalities on menu buttons for mobile version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://eddyshimwa.github.io/CapStone-Project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+### Install
+
+### Usage
+
+
+
+### Run tests
+
```

Comment:
*OPTIONAL*

- Please, let's remove the unused header section as this section are not relevant to this current project, you'd add them for the subsequent project so as to make your readme more professional.